spec/index.bs

```diff
@@ -1737,8 +1744,25 @@ conversion, with intermediate computations having at least the precision of the
 
 ### Color Space Conversion Elision ### {#color-space-conversion-elision}
 
-Issue: Write this section.
-
+If the source and destination of a color space/encoding conversion are the same, then conversion
+is not necessary. In general, if any given step of the conversion is an identity function (no-op),
+implementations **should** elide it, for performance.
```

Comment:
This reminds me that I wanted to ask again if we should consider updating the full spec to be consistent with [RFC2119](https://datatracker.ietf.org/doc/html/rfc2119) terminology. (MUST, SHOULD, MAY, etc...)

Reply:
Maybe! I think we mostly use those words correctly, but we have a lot of parts of the spec that don't use them at all, like "This thing does this" instead of "This thing must do this" and I don't know how to update those without making them difficult to read.